ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/VertxRequestTransmitter.java

```diff
@@ -57,7 +60,14 @@ public void sendRequest(
       final Iterable<Entry<String, String>> headers,
       final String path,
       final String body) {
-    LOG.debug("Sending request {} to {}", body, path);
+    if (LOG.isDebugEnabled()) {
```

Comment:
can we use the lambda use case for this (rather than a conditional?) - don't ask me how to do it though ...